src/netlink_namespace.c

```diff
@@ -688,18 +693,24 @@ static int ir_config(const char *key, const char *value) {
   return (status);
 } /* int ir_config */
 
-static void close_fds(int* selfns, int* targetns) {
-  if ((*selfns >= 0) && (close(*selfns) != 0)) {
-    ERROR("Closing \"selfns\" file failed: %m");
-  }
+static void ir_init_error(int* selfns, int* targetns) {
```

Comment:
I'm also slightly curious - why pointers-to-ints instead of just "plain ints" ?

The primary reason to use pointers would be to be able to modify the value the pointer is pointing to, at the cost of all the error-conditions around pointers (dealing with null-pointers and lifetime of things pointed to).

In this case - passing pointers is even more expensive, you're paying an extra instruction to de-reference the pointer rather than just dealing with the value.